salt/docker/init.sls

```diff
@@ -16,6 +16,7 @@ include:
         Environment="NO_PROXY={{ salt['pillar.get']('proxy:no_proxy', '') }}"
   cmd.run:
     - name: systemctl daemon-reload
+    - stateful: True
```

Comment:
This is still a WIP, that's why I didn't ask for review yet :), but in this context it will perfectly work, since no output is considered as "nothing changed" given the protocol definition, what perfectly fits in the context of a `daemon-reload`, IMO.